go/libkb/env.go

```diff
@@ -1016,7 +1016,10 @@ func (e *Env) GetMountDir() (string, error) {
 		if runMode != ProductionRunMode {
 			return "", fmt.Errorf("KBFS is currently only supported in production mode on Windows")
 		}
-		return "k:", nil
+		return e.GetString(
```

Comment:
in general, i like to have the consistency of having everything available via config and command line parameters. Can you do that here too?  See this file for plenty of examples